server/flyway/sql/V39__add_client_role.sql

```diff
@@ -0,0 +1,47 @@
+-- Delete any role assignments to the CLIENT initial testing roles
+DELETE FROM app_fam.fam_user_role_xref WHERE role_id IN
+    (SELECT role_id from app_fam.fam_role WHERE application_id IN
+        (SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD'))
+    )
+;
+
+-- Delete previously created CLIENT testing roles for client_dev, client_test and client_prod applications
+DELETE FROM app_fam.fam_role WHERE application_id IN (
+    SELECT application_id FROM app_fam.fam_application WHERE application_name IN ('CLIENT_DEV', 'CLIENT_TEST', 'CLIENT_PROD')
+);
+
+-- Add a role for CLIENT_DEV
+INSERT INTO app_fam.fam_role (
+    role_name,
+    role_purpose,
+    application_id,
+    role_type_code,
+    create_user,
+    create_date
+)
+VALUES ('CLIENT_EDITOR', 'Create and edit client information', (select application_id from app_fam.fam_application where application_name = 'CLIENT_DEV'), 'C', CURRENT_USER, CURRENT_DATE)
```

Comment:
This could be a better description

Reply:
E.g. Ministry role to approve/reject submissions, create client records, and perform non-administrative edits to client records.

Reply:
yeah I asked Ziad to provide a description sentence instead of an ability list (that's too long to put), and he said he just wants this simple one.....